www/docs/nextjs/ssg.md

```diff
@@ -14,41 +14,83 @@ Static site generation requires executing tRPC queries inside `getStaticProps` o
 ## Fetch data in `getStaticProps`
 
 ```tsx
-import Head from 'next/head';
-import { trpc } from '../utils/trpc';
+/**
+ * Given we're this file something like `pages/posts/[id].tsx`
+ */
 import { createSSGHelpers } from '@trpc/react/ssg';
+import {
+  GetStaticPaths,
+  GetStaticPropsContext,
+  InferGetStaticPropsType,
+} from 'next';
+import { prisma } from 'server/context';
+import { appRouter } from 'server/routers/_app';
+import superjson from 'superjson';
+import { trpc } from 'utils/trpc';
 
-// Statically fetch the data in getStaticProps
-export const getStaticProps = async (
-  context: GetStaticPropsContext<{ filter: string }>,
-) => {
-  const ssg = createSSGHelpers({
+
+export async function getStaticProps(
```

Comment:
Nit: should this go after `getStaticPaths`? Since the paths param is an input to it.